api/v1alpha3/docluster_types.go

```diff
@@ -29,15 +29,22 @@ const (
 
 // DOClusterSpec defines the desired state of DOCluster.
 type DOClusterSpec struct {
-	// The DigitalOcean Region the cluster lives in.
-	// It must be one of available region on DigitalOcean. See https://developers.digitalocean.com/documentation/v2/#list-all-regions
+	// The DigitalOcean Region the cluster lives in. It must be one of available
+	// region on DigitalOcean. See
+	// https://developers.digitalocean.com/documentation/v2/#list-all-regions
 	Region string `json:"region"`
 	// Network configurations
 	// +optional
 	Network DONetwork `json:"network,omitempty"`
-	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
+	// ControlPlaneEndpoint represents the endpoint used to communicate with the
+	// control plane. If ControlPlaneDNSRecord is unset, the DO load-balancer IP
+	// of the KAS is used.
 	// +optional
 	ControlPlaneEndpoint clusterv1.APIEndpoint `json:"controlPlaneEndpoint"`
+	// ControlPlaneDNSRecord is a managed DNS record that points to the
+	// load-balancer IP used for the ControlPlaneEndpoint.
+	// +optional
+	ControlPlaneDNSRecord *DOControlPlaneDNSRecord `json:"controlPlaneDNSRecord"`
```

Comment:
I would prefer to use `ControlPlaneDNS` for naming here.
Because saw `Record` I was thought, I also need to configure the whole DNS record such as type ('A', 'CNAME', etc), TTL here.
This is just a comment, If you don't think so just ignore it 😀

Reply:
SGTM, done.